openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/Graph.vue

```diff
@@ -1784,6 +1784,15 @@ export default {
           }
         }
       }
+
+      // Enforce pointsSaved limit - remove oldest data points if exceeded
```

Comment:
This needs hysterisis such that every new sample doesn't require rebuilding the array.   
Basically check from setting + 100 data points, and then do the cut back to the limit.